app/helpers/open_graph_helper.rb

```diff
@@ -1,15 +1,15 @@
 module OpenGraphHelper
   require "addressable/uri"
 
-  def opengraph_tags(title = nil, og_image = nil, og_image_alt = nil)
+  def opengraph_tags(title = nil, properties = {})
```

Comment:
This helper is only called in one place in the codebase, and it is called with two parameters. So the defaults never apply. They can be omitted entirely. 

Even if you call it as `opengraph_tags(nil, nil)` (or things that evaluate to nil, like an undefined instance variable) then `properties` will be `nil`, not `{}`.

Reply:
removed default values